spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -1322,7 +1313,7 @@ def get_connections_from_machine(
             transceiver, placement, master_pop_table, indirect_synapses,
             direct_synapses, key, pre_vertex_slice.n_atoms, index,
             using_extra_monitor_cores, placements, monitor_api,
-            extra_monitor, monitor_cores, fixed_routes=fixed_routes)
```

Comment:
extra_monitor should still be in the argument list here

Reply:
confused. everywhere else it was removing 2 elements. why would this one only need to remove 1? also wheres the handle_time_out_config? ug. 

but the tests pass.......... very very confused

Reply:
The integration tests don't pass - check the equivalent spynnaker8 branch...

Reply:
doh. tis good to renember theres a 8 at the end of that repo. ;-)

Reply:
good catch by the way on the bug